### ISSUE TITLE
Add option to show current playback speed

### ISSUE THREAD

**Issue by CaptainFlint** [smplayer-dev/smplayer#305]
When you open a file, it's not clear which speed it's playing at. If it was played before at a different speed, the player remembers it and sets the speed to what it was. Which is fine and expected, but if you already forgot about changing speed, it may take you by surprise. Currently, the only way to see the speed I could find, is to change it, say, +10%, and back -10%. The OCD and statusbar will then show the new speed value for a few seconds. But this is not as convenient as being able to just see it from the start.

Possible solutions I could think of:

1. Add another option to the Options-Statusbar submenu to always show the current playback speed in the statusbar.

2. Modify the "speed_menu" button so that instead of an icon it displayed the current speed value (this is what VLC has, for example, see the attached screenshot). Or, maybe, have two identical buttons: one with icon, another with current speed, so that users could choose what they prefer to see in the interface.

The screenshot of VLC's speed indicator, and the speed change popup which appears when clicking on it:
![3ftRnFLT7O](https://user-images.githubusercontent.com/6475044/145238544-f18773ca-b6c4-428c-ad75-c8904c638cc5.png)

**Comment by zootboy** [comment]
I don't think this should be auto-closed. It's a pretty self-explanatory feature request that's still valid.

**Comment by CaptainFlint** [comment]
Comment for un-staling.

**Comment by dk0830** [comment]
i'd like to see this feature as well. very important to me. i have been a user of sm player for over 10y 
thanks

**Comment by dk0830** [comment]
we need this feature. bump 07-24-2026
